ambry-commons/src/main/java/com.github.ambry.commons/BlobId.java

```diff
@@ -87,13 +87,15 @@
   private static final short DATACENTER_ID_FIELD_LENGTH_IN_BYTES = Byte.BYTES;
```

Comment:
Could you add information about the flag bits to the doc above the class? ^

Reply:
added.